docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,66 @@
+
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of projects. They offer a foundation for customization and serve as versatile building blocks.
```

Comment:
I think the terminology of this document is not aligned with the public terminology, eg: "core", "project", "customer", "SCOS", "core quality", "project quality"

Reply:
OK, then let's use the React API example, because the Blog Example does not really fit the definition of "nonreuseable buisness logic" modules very well. All of the better example modules are new and mostly related to unified commerce.

My original idea was to add a list of modules, but this needs to be done after we fixed the readmes and license files for the old modules.

Reply:
What is the public terminology I have to use here? 😕

Reply:
@gerner-spryker I've asked @lenadoc and she doesn't know either what exactly you want to change here.

Reply:
@floriankraemer Are the above listed terms well-known and generally used by the audience of docs.spryker.com and Lena?